backend/.env.docker

```
@@ -1,7 +1,7 @@
 MONGODB_URI="mongodb://mongo-service/wanderlust"
 REDIS_URL="redis://redis-service:6379"
 PORT=8080
-FRONTEND_URL="http://18.224.72.54:5173"
+FRONTEND_URL="http://192.168.192.146:5173"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)